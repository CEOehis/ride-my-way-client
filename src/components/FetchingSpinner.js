import React from 'react';

const FetchingSpinner = () => (
  <i style={{ textAlign: 'center', display: 'block' }} className="fa fa-spinner fa-spin fa-pulse fa-5x" aria-hidden="true" />
);

export default FetchingSpinner;
