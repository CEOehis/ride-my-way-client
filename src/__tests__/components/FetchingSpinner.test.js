import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import FetchingSpinner from '../../components/FetchingSpinner';

describe('Form submit FetchingSpinner component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<FetchingSpinner submitting />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
