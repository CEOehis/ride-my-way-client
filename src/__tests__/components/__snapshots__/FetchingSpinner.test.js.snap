// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form submit FetchingSpinner component should render correctly 1`] = `
<i
  aria-hidden="true"
  className="fa fa-spinner fa-spin fa-pulse fa-5x"
  style={
    Object {
      "display": "block",
      "textAlign": "center",
    }
  }
/>
`;
